src/controllers/SlideController.ts

```diff
@@ -1,10 +1,27 @@
 import { NextFunction, Response } from 'express'
+import { SlideRequestInterface } from '../interfaces/UserInterface'
 import { CreateSlideRequest } from '../interfaces/SlideInterface'
 import SlideService from '../services/SlideService'
 import { slideSchema } from '../validation/slideValidation'
 import { validator } from '../validation/validator'
 import GenericError from '../helpers/GenericError'
 
+const deleteSlide = async (
+  req: SlideRequestInterface,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    await SlideService.deleteSlide(req.params.id)
+    res.status(204).json({ message: 'success' })
+  } catch (error) {
+    const exception = error as Error
+
+    if (exception.name !== 'GenericError') return next(exception)
+    return res.status(204).send({ error: exception.message })
```

Comment:
no need, the delete slide service won't throw any generic error so just `next(exception)`

Reply:
sorry, 
for these disastrous mistakes and sorry for saying l will do it as soon as possible as I can